chimney/src/main/scala/io/scalaland/chimney/internal/TransformerCfg.scala

```diff
@@ -54,7 +56,8 @@ trait TransformerConfiguration extends MacroUtils {
       definitionScope: Option[(Type, Type)] = None,
       wrapperType: Option[Type] = None,
       wrapperSupportInstance: Tree = EmptyTree,
-      coproductInstancesF: Set[(Symbol, Type)] = Set.empty // pair: inst type, target type
+      coproductInstancesF: Set[(Symbol, Type)] = Set.empty, // pair: inst type, target type
+      namingConventions: (String => String, String => String) = (identity, identity) // Function to apply on the From accessor to get it in the To convention
```

Comment:
Wouldn't be better to create separate interface/class for it? Two `String => String` functions are a bit cryptic IMO.